api/src/main/java/org/accula/api/handlers/response/ResponseBody.java

```diff
@@ -0,0 +1,15 @@
+package org.accula.api.handlers.response;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.ANY;
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+
+/**
+ * @author Anton Lamtev
+ */
+@JsonInclude(NON_NULL)
+@JsonAutoDetect(fieldVisibility = ANY)
+public interface ResponseBody {
```

Comment:
I would suggest to rename with a more concrete name, `NonNullFieldsJsonResponseBody` for example

Reply:
__-__